test/unit/cli.test.ts

```diff
@@ -38,6 +38,27 @@ describe('cli', (): void => {
         SHOW_HIDDEN: false,
         COLOR: true
       },
+      DATABASE: {
+        ACQUIRE_TIMEOUT_MILLIS: 30000,
+        CREATE_RETRY_INTERVAL_MILLIS: 200,
+        CREATE_TIMEOUT_MILLIS: 30000,
+        DEBUG: false,
+        DESTROY_TIMEOUT_MILLIS: 5000,
+        DIALECT: 'mysql',
```

Comment:
Shouldn't this be `sqlite`? In other words, do tests still pass if you turn off your local MySQL database?

Reply:
@spikerheado1234 : Thoughts?